src/main/java/io/openshift/launchpad/ui/osio/BoosterDTO.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Eclipse Public License version 1.0, available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package io.openshift.launchpad.ui.osio;
+
+import io.openshift.booster.catalog.Booster;
+import io.openshift.booster.catalog.Mission;
+import io.openshift.booster.catalog.Runtime;
+
+/**
+ */
+public class BoosterDTO
+{
+   private String id;
+   private String name;
+   private String description;
+   private String missionId;
+   private String missionName;
+   private String runtimeId;
+   private String runtimeName;
+
+   public BoosterDTO()
+   {
+   }
+
+   public BoosterDTO(Booster booster)
+   {
+      this.id = booster.getId();
+      this.name = booster.getName();
+      this.description = booster.getDescription();
```

Comment:
Add the Booster object as part of this DTO and add the `@java.beans.Transient` annotation in the getter method to prevent serialization